packages/library-legacy/src/connection.ts

```diff
@@ -3724,7 +3724,17 @@ export class Connection {
     commitment?: Commitment,
   ): Promise<RpcResponseAndContext<SignatureResult>>;
 
-  /** @deprecated Instead, call `confirmTransaction` and pass in {@link TransactionConfirmationStrategy} */
+  /** @deprecated Instead, call `confirmTransaction` and pass in {@link TransactionConfirmationStrategy}
```

Comment:
This should get moved upward to the non-deprecated overload.

Reply:
roger that. I added it there because I want to help people that come across old documentation.
The typical path is that `anchor test` fails because the airdrop transaction was not confirmed yet.
We could address the root cause by pointing people to this function:
``` ts
const newBalance = await airdropIfRequired(
  connection,
  keypair.publicKey,
  0.5 * LAMPORTS_PER_SOL,
  1 * LAMPORTS_PER_SOL,
);
```
found here: https://www.npmjs.com/package/@solana-developers/helpers

this package also contains the confirmTransaction function that was deprecated here.

Reply:
This doesn't look like the deprecated version to me.

https://github.com/solana-developers/helpers/blob/f33977badf111033930663237de76552a3dd93c1/src/index.ts#L266-L271